### PR TITLE
survey displays if has not been submitted

### LINKED ISSUE #215: Redirect to Symptom Tracker Survey when user login
## Summary

Add functionality to display survey on login, if survey hasn't yet been completed for the day

## Motivation

We want people to submit the survey every day, and visualizing it when they didn't submit it yet will help us remind people to do so.

### PR DESCRIPTION
**Pull request by NGabuaeva**
## IMPORTANT: Please do not create a Pull Request without creating an issue first.

**Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.**

**Do not change the following line aside from replacing YOUR-ISSUE-HERE with the issue this PR fixes** 

fixes #215 

### Explain the **details** for making this change. What existing problem does the pull request solve?
If the survey hasn't been submitted for the day, after logging in survey page will be rendered, otherwise the home page will be displayed. I am using window.localStorage to solve this problem, can add the code for it for browsers that don't support localStorage if necessary. 
### Test plan (required)

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.
Ran the app locally two days in a row, if the survey has not been submitted the first page the user will see after logging in will be the survey. If the user logs out and logs back in the same day the home page will be displayed.

### Final Checklist

- [ ] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [ ] Have you added any new tests necessary?
- [ ] Is your PR rebased off the most current master?
- [ ] Have you squashed all commits, or if you will merge will you squash all commits?
- [ ] Did you use yarn, not npm?
- [ ] Did you use Material-UI wherever possible?
- [ ] Did you format according to Prettier?
- [ ] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: client/src/components/DiagnosticContainer.jsx [hunk 1]
```diff
@@ -23,31 +24,49 @@ const dateOptions = {
   day: 'numeric',
 };
 
+// check if the survey has been submitted today
+const hasSubmitted = () => {
+  const date = window.localStorage.getItem('date');
+  const todaysDate = new Date().toISOString().slice(0, 10);
+  if (date === todaysDate) {
```
Comment (SomeMoosery) [inline client/src/components/DiagnosticContainer.jsx:31]: Would there be a case where these two aren't equal?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (SomeMoosery) [inline yarn.lock:4]: Please delete this `yarn.lock` in the root directory

File: client/src/components/DiagnosticContainer.jsx [hunk 3]
```diff
@@ -23,31 +24,49 @@ const dateOptions = {
...
+    if (window.localStorage.getItem('surveyCompleted') === 'false') {
+      return false;
+    }
+    return true;
+  }
+  window.localStorage.setItem('date', todaysDate);
+  window.localStorage.setItem('surveyCompleted', 'false');
```
Comment (SomeMoosery) [inline client/src/components/DiagnosticContainer.jsx:38]: Is there any benefit of using local storage here and not storing the surveyCompleted state in the Redux store?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -378,33 +378,33 @@ Unsure where to begin contributing to Atom? You can start by looking through the
...
 
-See the full list of commands [here](https://classic.yarnpkg.com/en/docs/cli/).  
+See the full list of commands [here](https://classic.yarnpkg.com/en/docs/cli/).
 
-`yarn install`  
-Installs all node modules  
-Similar to `npm i`  
+`yarn install`
+Installs all node modules
+Similar to `npm i`
```
Comment (SomeMoosery) [inline CONTRIBUTING.md:385]: These changes shouldn't be here as it adds unnecessary file changes to the PR

File: client/src/components/DiagnosticContainer.jsx (hunk 1 above)
Comment (NGabuaeva) [reply client/src/components/DiagnosticContainer.jsx:31]: Yes. If I am logging in today today's date will be stored in the local storage, so when I login again tomorrow they will not be equals, so the function will return false and I will be shown the survey after logging in

File: client/src/components/DiagnosticContainer.jsx (hunk 3 above)
Comment (NGabuaeva) [reply client/src/components/DiagnosticContainer.jsx:38]: My thinking was that the localStorage is more persistent, so if the user opened the app in the morning, filled out the survey, then logged out, closed the app, later when the user opens the app again the same day, the survey won't be displayed after logging in. This way we can make sure that the user gets the survey as the first page only once a day

**Comment by NGabuaeva** [comment]
Before login
After login for the first time today 2020-04-03 at 12 52 42 PM
After logging in again the same day 2020-04-03 at 12 53 56 PM

Let me know if these work

File: yarn.lock (hunk 2 above)
Comment (NGabuaeva) [reply yarn.lock]: Deleted in the last commit

File: CONTRIBUTING.md (hunk 4 above)
Comment (NGabuaeva) [reply CONTRIBUTING.md]: I haven't touches this file, not sure why it even showed up in changes, but should be fixed in the last commit

**Comment by NGabuaeva** [comment]
Could you tell me about the rebase off of master? I pulled from upstream, is there more I need to do?

File: client/src/components/DiagnosticContainer.jsx (hunk 3 above)
Comment (SomeMoosery) [reply client/src/components/DiagnosticContainer.jsx:38]: Makes sense 👍

**Comment by SomeMoosery** [comment]
All changes looking good me now, thanks for the explanations! It looks like you've rebased properly as well - there's some guides in our CONTRIBUTING.md around it for future reference but it looks like you did it properly here.

One more thing I forgot to mention and then this is good to go - if you could just bump the `version` in `package.json` in the client, that would be great!

**Comment by NGabuaeva** [comment]
Bumped the version from 0.2.10 to 0.2.11, is that correct?

**Comment by SomeMoosery** [comment]
Yup, exactly!

**Pull request merged**